cloud/db_cloud_test.cc

```diff
@@ -373,7 +368,7 @@ TEST_F(CloudTest, Newdb) {
     // buckets as newdb1. This should be identical in contents with newdb1.
     std::unique_ptr<CloudEnv> cloud_env;
     std::unique_ptr<DBCloud> cloud_db;
-    CloneDB("newdb2", src_bucket_prefix_, src_object_prefix_, "", "", &cloud_db,
+    CloneDB("newdb2", "", "", &cloud_db,
```

Comment:
Is this correct? Previously we supplied `src_bucket_prefix_` and `src_object_prefix_` here and now it's empty strings.

Reply:
The src_XXX options were removed, as they were always being passed in as the old values.